README.md

```diff
@@ -1,2 +1,59 @@
 # Salesforce-Postman
 Salesforce API Postman Collection
+
+# How to install it
+
+Go to the [release section](https://github.com/scolladon/Salesforce-Postman/releases) to download the salesforce-apis.postman_export.json from the release you want.
+
+[Import](https://learning.postman.com/docs/postman/collections/data-formats/#importing-postman-data) it into postman
+
+# How to contribute
+## Prerequisite
+You need to have installed :
+- [Postman](https://learning.postman.com/docs/postman/launching-postman/installation-and-updates/).
+- [Postman Extractor](https://github.com/pozil/postman-extractor).
+
+## Development Lifecycle
```

Comment:
Let's separate this in two sections:
- Using the Collection
- Contributing

Reply:
In the Using section, list the basic steps:
1. Configuring the environment
2. Authenticating
3. Calling an API endpoint